config/env.production

```diff
@@ -165,6 +165,17 @@ PARSER_URL=https://api.telescope.cdot.systems/v1/parser
 PLANET_PORT=9876
 
 
+################################################################################
+# Docusaurus App
+################################################################################
+
+# Docusaurus Server Port (default is 4631)
+DOCS_PORT=4631
```

Comment:
We won't run this server in staging/production, right?  Can this get removed?

Reply:
Removed